client/src/charts/progressGauge.scss

```diff
@@ -0,0 +1,11 @@
+.progress-circle {
+    display: inline-block;
+    height: 2rem;
+    width: 2rem;
+    margin: 0 1rem;
+    border-radius: 100%;
+}
+
+.empty {
```

Comment:
let's try to use more specific names for everything instead of generic names, unless it's really something generic